.travis.yml

```diff
@@ -8,8 +8,27 @@ perl:
   - "5.14"
   - "5.12"
   - "5.10"
+matrix:
+  include:
+    - perl: 5.22
+      env: COVERAGE=1   # enables coverage+coveralls reporting
+      include:
+    - perl: 5.24
+      env: COVERAGE=1   # enables coverage+coveralls reporting
+sudo: false
 before_install:
   - git clone git://github.com/travis-perl/helpers ~/travis-perl-helpers
   - source ~/travis-perl-helpers/init
   - build-perl
   - perl -V
+  - build-dist
+  - cd $BUILD_DIR             # $BUILD_DIR is set by the build-dist command
+install:
+  - cpan-install ExtUtils::MakeMaker~6.68 --deps
```

Comment:
Why do we need this specific version of ExtUtils::MakeMaker?  I just see a comment about cargo culting, but it's not clear to me from where or why.